gradio/data_classes.py

```diff
@@ -227,6 +229,19 @@ class FileData(GradioModel):
     is_stream: bool = False
     meta: dict = {"_type": "gradio.FileData"}
 
+    @model_validator(mode="before")
+    @classmethod
+    def validate_model(cls, v, info: ValidationInfo):
+        if (
+            info.context
+            and info.context.get("validate_meta")
+            and v.get("meta", {}) != {"_type": "gradio.FileData"}
```

Comment:
is it possible to use `client_utils.is_file_obj_with_meta` directly here in order to prevent any discrepancies in the future?

Reply:
Yes!